src/Raven.Studio/typings/_studio/dto.d.ts

```diff
@@ -643,3 +643,7 @@ interface rawStackTraceResponseItem {
 }
 
 type indexStatus = "Normal" | "ErrorOrFaulty" | "Stale" | "Paused" | "Disabled" | "Idle";
+
+interface Configuration {
```

Comment:
Can we rename this something less generic, like MigratorPathConfiguration

Reply:
done (my original thought was that maybe 'configuration' interface will represent other settings in the future)

Reply:
but this is migrator specific